sriov_ib_cni_test.sh

```diff
@@ -133,6 +136,10 @@ function test_guids_reseted {
     done
 }
 
+function get_vfs_guids {
```

Comment:
you are only doing ip link show on `ib0`, should this be a parameter for the function ?

Reply:
totally, will do it so that it gets this automatically, (the logic is already implemented in the install script, will do it here also)

Reply:
This is done, but i think because i did not change this same line, it is not marked as outdated.